tailcfg/tailcfg.go

```diff
@@ -2401,6 +2396,14 @@ const (
 	// NodeAttrSSHEnvironmentVariables enables logic for handling environment variables sent
 	// via SendEnv in the SSH server and applying them to the SSH session.
 	NodeAttrSSHEnvironmentVariables NodeCapability = "ssh-env-vars"
+
+	// NodeAttrServiceHost grants a peer the ability to serve as a host for a
```

Comment:
It's not granting a peer abilities (that would be a `PeerCapability`), it's communicating configuration details for a service being advertised.

Maybe "NodeAttrServiceHost indicates the VIP Services for which the client is approved to act as a service host, and which IP addresses are assigned to those VIP Services. Any VIP Services that the client is not advertising can be ignored. Each value of this key in [NodeCapMap] is of type [ServiceIPMappings]. If multiple values of this key exist, they should be merged in sequence."

Reply:
oops, that's a leftover from the typo. thanks!